swagger.yaml

```diff
@@ -30,6 +26,22 @@ paths:
           schema:
             $ref: '#/definitions/Info'
 
+  /departments:
+    get:
+      summary: List departments
+      description: Return org code and org description (departments) for a business center
+      parameters:
+        - in: query
```

Comment:
Does the api accept multiple business_center parameters? If so, what is the collection format?